### ISSUE TITLE
Cinny shouldn't autoplay animated avatars

### ISSUE THREAD

**Issue by tezlm** [ajbura/cinny#1159]
### Describe the problem

Cinny plays animated avatars (gif or webp) at all times, which is distracting.

### Describe the solution you'd like

Cinny shouldn't play animated avatars unless on hover or if they're the main focus of something (eg. room member popup)

[MSC2705](https://github.com/matrix-org/matrix-spec-proposals/blob/travis/msc/animated-thumbnails/proposals/2705-thumbnail-requirements.md) could be used to thumbnail images. This is still a msc but seems to be supported among media repo implementations that support animated thumbnails.

It also looks like src/app/atoms/avatar/render.js has logic for rendering to a canvas, which would remove animation. However, this would take an extra parsing/rendering an avatar when its first loaded and would need extra memory to store the parsed avatars (which can't be cached on disk).

**Comment by viasux** [comment]
> MSC2705

I completely disagree. They should be (at least by default) played while the browser/app is focused. Pausing while out of focus is reasonable though.

**Comment by tezlm** [comment]
An entire page full of small animated avatars is extremely distracting and borderline unusable. Discord also doesn't play animated avatars until you click on it. Avatars, unlike stickers and gifs, are visual identifiers for users and should draw as little attention as possible. For people who are sure they do want this, "autoplay all animated avatars" could be an appearance (accessibility?) option.

**Comment by viasux** [comment]
> An entire page full of small animated avatars is extremely distracting and borderline unusable. Discord also doesn't play animated avatars until you click on it. Avatars, unlike stickers and gifs, are visual identifiers for users and should draw as little attention as possible. For people who are sure they do want this, "autoplay all animated avatars" could be an appearance (accessibility?) option.

I completely understand that you have trouble looking at animated profile pictures, however, most people probably don't, thus I believe it should at least be the default, as it represents users how they wish to be represented. Someone with a gif pfp doesn't want to be represented by a single frame of that gif 90% of the time, unless someone hovers over their profile.

Like I said, i wouldn't mind if it was an option

**Comment by tezlm** [comment]
If people want to be represented, this doesn't change that. Other people can view it in a low friction way by viewing their profile, or in an even lower friction way by hovering over the pfp. I have never once seen anyone want to have all avatars animated until yesterday.

Again, it should absolutely be an option. I hope we can agree to disagree.

**Comment by viasux** [comment]
I have never once seen anyone want to have all avatars not animated until yesterday.

**Comment by hopolapopola** [comment]
"never once [seeing] anyone want to have" an accessibility feature doesn't mean there aren't people who want or need said feature. This is a no-brainer to implement and should at least be toggleable

**Comment by viasux** [comment]
> "never once [seeing] anyone want to have" an accessibility feature doesn't mean there aren't people who want or need said feature. This is a no-brainer to implement and should at least be toggleable

yes, but it shouldn't be default, "reduce motion/etc." exists for a reason
